web/src/components/HeaderActionItemDropDown.tsx

```diff
@@ -11,19 +13,44 @@ const StyledIcon = styled(Icon)`
   width: 28px;
   height: 28px;
 `
+const ActionBox = styled(Button)<{ $isRTL?: boolean }>`
+  height: 36px;
+  min-width: 122px;
+  display: flex;
+  border-color: black;
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 4px;
+  justify-content: center;
+  align-items: center;
+  flex-direction: ${props => (props.$isRTL ? 'row-reverse' : 'row')};
```

Comment:
You can just use the following, no need to pass this value:
```suggestion
  flex-direction: ${props => (props.theme.contentDirection === 'rtl' ? 'row-reverse' : 'row')};
```